src/HealthChecks.Uris/UriHealthCheckOptions.cs

```diff
@@ -12,6 +13,8 @@ public interface IUriOptions
         IUriOptions UseTimeout(TimeSpan timeout);
         IUriOptions ExpectHttpCode(int codeToExpect);
         IUriOptions ExpectHttpCodes(int minCodeToExpect, int maxCodeToExpect);
+        IUriOptions ExpectContent(string contentToExpect);
+        IUriOptions ExpectContent(Func<HttpContent, Task<(bool IsOk, string NotOkReason)>> contentCheckerFunc);

```

Comment:
Same here, use Func of string instead tuples

Reply:
See above.